packages/forms/src/widgets/ColumnsWidget/__snapshots__/ColumnsWidget.test.js.snap

```diff
@@ -11,6 +11,10 @@ ShallowWrapper {
   "node": <div
     className="tf-widget-columns undefined"
 >
+    <TitleField
```

Comment:
Can you snapshot `wrapper.node` instead ? The snapshot would be less big and more readable